tensorflow/core/platform/cloud/retrying_file_system.cc

```diff
@@ -40,20 +42,22 @@ bool IsRetriable(Status status) {
 Status CallWithRetries(const std::function<Status()>& f) {
   int attempts = 0;
   while (true) {
+    int delay = std::min(initial_delay_seconds << attempts, kMaximumBackoffSeconds);
```

Comment:
Minor comments: 
1) could you please add a 'const' modifier?
It's recommended by the style guide, see https://google.github.io/styleguide/cppguide.html#Use_of_const for details. 

2) Could you also move this line to right before line 53, since 'delay' is not used anywhere besides the last line in the loop. This should make the code more readable.